src/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -622,7 +622,7 @@ public override string ToString()
                     return true;
                 }
 
-                if (ReferenceEquals(left.name, null) || ReferenceEquals(right.name, null))
+                if (ReferenceEquals(left.name, null) | ReferenceEquals(right.name, null))
```

Comment:
Do you have perf tests to show this in a net win across all of the various cases?  If it's not measurable, I'd prefer to leave it as it was.  If it is measurable, it'd be good to validate the change moves it in the right direction.